.github/workflows/publish-container.yaml

```diff
@@ -16,41 +16,62 @@ jobs:
-            type=ref,event=pr
-            type=semver,pattern={{version}}
-            type=sha,format=long
-      - name: Get committer date
+      - name: Setup and configure Docker
+        shell: 'script -q -e -c "bash {0}"'
         run: |
-          echo "committer_date=$(git log -1 --pretty=%ct)" >> "$GITHUB_ENV"
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # v3.3.0
+          [[ $EUID -ne 0 ]] && exec sudo /bin/sh "$0" "$@"
+          for pkg in docker.io docker-doc docker-compose docker-compose-v2 podman-docker containerd runc; \
+            do apt-get remove $pkg; \
+          done
+          apt-get update
+          apt-get install ca-certificates curl
+          install -m 0755 -d /etc/apt/keyrings
+          curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+          chmod a+r /etc/apt/keyrings/docker.asc
+          echo \
+            "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+            $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+            tee /etc/apt/sources.list.d/docker.list > /dev/null
+          apt-get update
+          cat << EOF >/etc/docker/daemon.json
+          {
+            "features": {
+            "containerd-snapshotter": true
+            },
+            "registry-mirrors": ["https://ghcr.io/tkhq"]
+          }
+          EOF
+          apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+          systemctl restart docker
+          docker buildx create --driver docker-container --bootstrap --name build --use
       - name: Login to GHCR
         if: github.event_name != 'pull_request'
         uses: docker/login-action@0d4c9c5ea7693da7b068278f7b52bda2a190a446 # v3.2.0
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Build and push
-        uses: docker/build-push-action@ca052bb54ab0790a636c9b5f226502c73d547a25 # v5.4.0
-        with:
-          context: .
-          platforms: linux/amd64,linux/arm64
-          build-args: |
-            SOURCE_DATE_EPOCH=${{ env.committer_date }}
-          provenance: "false"
-          push: ${{ github.event_name != 'pull_request' }}
-          tags: ${{ steps.meta.outputs.tags }}
+      - name: Build
+        shell: 'script -q -e -c "bash {0}"'
+        run: |
+          make
+      - name: Push
+        shell: 'script -q -e -c "bash {0}"'
+        run: |
+          env -C out/ecr-proxy tar -cf - . | docker load
+          docker tag "tkhq/ecr-proxy:latest" "ghcr.io/tkhq/ecr-proxy:latest"
+          for tag in ${tags}; do
+              docker tag "tkhq/ecr-proxy:latest" "ghcr.io/tkhq/ecr-proxy:${tag}"
+          done
+          docker image push --all-tags "ghcr.io/tkhq/ecr-proxy"
```

Comment:
Need to login before you push.

Reply:
We do the "login to ghcr" step with some module, but it appears not do anything, so will drop it.